dmoj/executors/base_executor.py

```diff
@@ -145,11 +145,11 @@ def run_self_test(cls, sandbox=True, output=True, error_callback=None):
             return False
 
     @classmethod
-    def get_versionable_commands(cls):
+    def get_versionable_commands(cls) -> Tuple[str, str]:
         return (cls.command, cls.get_command()),
```

Comment:
I think as per https://github.com/python/mypy/issues/184 this is `Tuple[Tuple[str, str], ...]`.